src/frontend/apps/deposit/components/Dashboard/DashboardStudent/UploadFiles/index.tsx

```diff
@@ -0,0 +1,201 @@
+import { Box, Button, Grid, Paragraph, Text } from 'grommet';
+import Dropzone from 'react-dropzone';
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage, useIntl } from 'react-intl';
+import { DateTime } from 'luxon';
+
+import { PlusSVG } from 'components/SVGIcons/PlusSVG';
+import { UploadableObjectProgress } from 'components/UploadableObjectProgress';
+import {
+  UploadManagerStatus,
+  useUploadManager,
+} from 'components/UploadManager';
+
+import { depositAppData } from 'apps/deposit/data/depositAppData';
+import { useDepositedFiles } from 'apps/deposit/data/queries';
+import { createDepositedFile } from 'apps/deposit/data/sideEffects/createDepositedFile';
+import { modelName } from 'apps/deposit/types/models';
+import { bytesToSize } from 'apps/deposit/utils/bytesToSize';
+import { truncateFilename } from 'apps/deposit/utils/truncateFilename';
+
+const messages = {
+  dropzonePlaceholder: {
+    defaultMessage: "Drag 'n' drop some files here, or click to select files",
+    description: 'Placeholder for dropzone.',
+    id: 'apps.deposit.components.DashboardStudent.UploadFiles.dropzonePlaceholder',
+  },
+  uploadButtonLabel: {
+    defaultMessage: 'Upload',
+    description: 'Label for upload button.',
+    id: 'apps.deposit.components.DashboardStudent.UploadFiles.uploadButtonLabel',
+  },
+  uploadingFile: {
+    defaultMessage:
+      'Upload in progress... Please do not close or reload this page.',
+    description:
+      'Help text to warn user not to navigate away during file upload.',
+    id: 'apps.deposit.components.DashboardStudent.UploadFiles.uploadingFile',
+  },
+};
+
+export const UploadFiles = () => {
+  const intl = useIntl();
+  const { refetch: refreshDepositedFiles } = useDepositedFiles(
+    depositAppData.fileDepository!.id,
+    {},
+  );
+
+  const { addUpload, uploadManagerState } = useUploadManager();
+  const [filesToUpload, setFilesToUpload] = useState<File[]>([
+    // new File(['anrusitanrsui tnarsuit narsuit'], 'TestFile.txt'),
+    // new File(
+    //   ['anrusitanrsui tnarsuit narsuinrs manuris narusi nat'],
+    //   'TestFile2 TestFile2.txt',
+    // ),
+    // new File(
+    //   [
+    //     'anrusitanrsui tnarsuit narsuitanrusitanrsui tnarsuit narsuitanrusitanrsui tnarsuit narsuit',
+    //   ],
+    //   'TestFile3 TestFile3 TestFile3 TestFile3 TestFile3.txt',
+    // ),
```

Comment:
To remove ?

Reply:
I'll do if after rebasing.
I can still use it for debugging purpose.